raspberry/bzzz/sensors/time_of_flight_sensor.py

```diff
@@ -0,0 +1,206 @@
+from bzzz.sensors.filters import median_filter
+from time import time_ns, sleep
+from datetime import datetime
+import VL53L0X
+import os
+
+
+class TimeOfFlightSensor:
+    # NOTE: This class fetches altitude in millimeters and this class requires i2c interfacing with the sensor.
+    def __init__(self,
+                 num_latest_readings_to_keep: int = 3,
+                 use_sleep=False,
+                 cache_altitude=False,
+                 use_outlier_detection=False,
+                 abs_outlier_diff_thres=500):
+        """Time of flight sensor class initialization.
+
+        :param num_latest_readings_to_keep: Window length for median filter, defaults to 3
+        :param use_sleep: Indicator to use sleep and wait for the next measurement; Not recommended as this will halt the system, defaults to False
```

Comment:
Since `use_sleep` is not recommended and should not be used, why have it in the code?

Reply:
To be clear, I'm saying that we shouldn't have it in the code

Reply:
Removed!